docs/agents/synacc.rst

```diff
@@ -0,0 +1,82 @@
+.. highlight:: rst
+
+.. _synacc:
+
+==================
+SynAcc Agent
+==================
+The SynAcc Agent interfaces with the power strip over ethernet to control 
+different outlets as well as get their status. 
+
+.. argparse::
+    :filename: ../agents/synacc/synacc.py
+    :func: make_parser
+    :prog: python3 synacc.py
+
+
+Configuration File Examples
+---------------------------
+Below are configuration examples for the ocs config file and for running the
+Agent in a docker container.
+
+ocs-config
+``````````
+To configure the SynAcc Agent we need to add a SynAcc Agent block to our ocs 
+configuration file. Here is an example configuration block using all of the 
+available arguments::
+
+       {'agent-class': 'SynAccAgent',
+        'instance-id': 'synacc',
+        'arguments':[
+          ['--ip-address', '10.10.10.8'],
+          ['--username', 'admin'],
+          ['--password', 'admin'],
```

Comment:
How do these username/password get setup initially? I take it admin/admin is default?

Reply:
Yeah admin/admin is the default. You can change it in the web browser (it has a typical IP address site)